doc/PEDSnet_DQA_Documentation.md

```diff
@@ -76,5 +76,5 @@ generateSingleReport(level=2, report="table_name")
 "table_name" examples include: "procedure", "drug", "measurement", etc
 ```
 
-### Generate Github Issues
-To generate GitHub issues from Secondary Reports follow instructions [here](https://github.com/PEDSnet/Data-Quality-Analysis/blob/master/Tools/dqa/README.md).
+## Generate feedback (GitHub)
+To investigate differences between the previous data version and the current and generate feedback:[here](https://github.com/PEDSnet/Data-Quality-Analysis/blob/master/Tools/dqa/README.md).
```

Comment:
This sentence seems to be incomplete.
Also, the text is not showing up as a hyperlink